.github/workflows/sync-axelar-repo.yml

```diff
@@ -0,0 +1,35 @@
+name: Synchronize to private repo
+
+on:
+  workflow_dispatch:
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout current repo
+      uses: actions/checkout@v3
+      with:
+        token: ${{ secrets.PAT_TOKEN }}
+        fetch-depth: 0
+
+    - name: Sync with private repo
+      run: |
+
+
```

Comment:
there are extra spaces here for no reason ??

Reply:
removed